src/utils/__tests__/url-test.js

```diff
@@ -52,3 +58,95 @@ describe('getResource', () => {
     expect(resource).toEqual(expectedResult);
   });
 });
+
+describe('isUrlInAppLink', () => {
+  test('when link is external', () => {
+    expect(isUrlInAppLink('https://example.com', 'https://another.com')).toBeFalsy();
```

Comment:
You should likely use
```js
toBe(false);
```
to be precise. Are you expecting anything but a an actual `false` value?